README.md

```diff
@@ -55,10 +55,7 @@ computer.
 
 ## Localization
 
-This project is running with an external localization platform called POEditor. You can contribute localization by
-clicking the image below!
-
-[![Minicraft+ POEditor Stats](https://minicraft-plus-poeditor-stats.vercel.app/api/card)](https://minicraft-plus-poeditor-stats.vercel.app)
+This project is running with an external localization platform called Lokalise. You can contribute localization [here](https://app.lokalise.com/public/42925017655336d50b3007.03067253/)!
```

Comment:
This is bad hyperlink practice; https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#link_best_practices.

Reply:
Updated.